### PR TITLE
Refactor to grab org/group claim.

### PR DESCRIPTION
**Pull request by crashGoBoom**
Here are some of the changes to add the Group Membership client mapper named "org".

### CODE REVIEW

File: deploy/files.sql [hunk 1]
```diff
@@ -34,13 +39,14 @@ CREATE TABLE mappa.files (
   created_at  TIMESTAMPTZ,
   user_id     UUID,
   name        TEXT,
+  org         TEXT,
```
Comment (statik) [inline deploy/files.sql]: could you make this JSONB so that we can do structured queries against it later?

**Comment by crashGoBoom** [comment]
@statik Hi! Just made those changes. Let me know if you need anything else. Thanks!!

File: keycloak/data/havendev-realm.json [hunk 2]
```diff
@@ -1,1598 +1,1778 @@
...
+  ],
+  "clients": [
+    {
+      "id": "f292fcd9-4b25-4d58-bddc-9d29a72d7a59",
+      "clientId": "admin-cli",
+      "name": "${client_admin-cli}",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
```
Comment (statik) [inline keycloak/data/havendev-realm.json]: I'm not sure what to think about this line, previously it was "6cae6b16-4de1-4aa7-88b6-7dd0efa81b17"

File: keycloak/data/havendev-realm.json [hunk 3]
```diff
@@ -1,1598 +1,1778 @@
...
+      "attributes": {},
+      "realmRoles": [],
+      "clientRoles": {},
+      "subGroups": []
     },
-    "groups" : [ ]
-  }, {
-    "id" : "90920d91-3090-4b4a-ae2a-2377cfa06ecd",
-    "createdTimestamp" : 1492023544974,
-    "username" : "user1@havengrc.com",
```
Comment (statik) [inline keycloak/data/havendev-realm.json]: it seems like this user definition (along with the other two users) got dropped in the export)

File: README.md [hunk 4]
```diff
@@ -101,7 +101,7 @@ First enter the keycloak container
 
 Now run the keycloak server command with export strategies defined
 
-    /opt/jboss/keycloak/bin/standalone.sh -Dkeycloak.migration.action=export -Dkeycloak.migration.provider=dir -Dkeycloak.migration.dir=/keycloak -Dkeycloak.migration.usersExportStrategy=REALM_FILE -Dkeycloak.migration.realmName=havendev
+    /opt/jboss/keycloak/bin/standalone.sh -Dkeycloak.migration.action=export -Dkeycloak.migration.provider=singleFile -Dkeycloak.migration.file=havendev-realm.json -Djboss.http.port=8888 -Djboss.https.port=9999 -Djboss.management.http.port=7777
```
Comment (statik) [inline README.md]: thanks for figuring this out!

**Comment by statik** [comment]
👍 thank you!

**Pull request merged**